packages/core/src/discovery.js

```diff
@@ -195,16 +223,21 @@ async function* captureSnapshotResources(page, snapshot, options) {
   };
 
   // used to resize the using capture options
-  let resizePage = width => page.resize({
-    height: snapshot.minHeight,
-    deviceScaleFactor,
-    mobile,
-    width
-  });
+  let resizePage = width => {
+    page.network.currentWidth = width;
+    return page.resize({
+      height: snapshot.minHeight,
+      deviceScaleFactor,
+      mobile,
+      width
+    });
+  };
 
   // navigate to the url
   yield resizePage(snapshot.widths[0]);
-  yield page.goto(snapshot.url, { cookies });
+  let forceReload = false;
+  if (discovery.captureResponsiveAssetsEnabled) { forceReload = true; }
+  yield page.goto(snapshot.url, { cookies, forceReload });
```

Comment:
Optional
```suggestion
  yield page.goto(snapshot.url, { cookies, forceReload: discovery.captureResponsiveAssetsEnabled });
```

Reply:
@chinmay-browserstack  also account for the additional cookies which we are planning to capture via SDKs, how they are going to be passed here?

Reply:
we can pass cookies is same fashion as we currently do inside domSnapshot object. we will override this value